src/lib/settings/SchemaFolder.js

```diff
@@ -118,7 +118,7 @@ class SchemaFolder extends Schema {
 		if (this.gateway.sql) {
 			if (piece.type !== 'Folder' || piece.keyArray.length) {
 				await this.gateway.provider.addColumn(this.gateway.type, piece.type === 'Folder' ?
-					piece.getSQL() : piece.sql[1]);
+					piece.getSQL() : [piece.sql]);
```

Comment:
I already fixed this in another PR.

Reply:
yes but since the related PR is not merged and i dont want to have merge conflicts this is fine.